src/khoj/routers/api.py

```diff
@@ -665,7 +665,7 @@ def trigger_manual_job(
 
 @api.put("/automation", response_class=Response)
 @requires(["authenticated"])
-def edit_job(
+async def edit_job(
```

Comment:
When the automation update/create encounters an error, it gets stuck on the client-side in the `Saving...` state. We should handle the error case and inform the user it didn't work.

Reply:
Good point. Updated the automation cards to show error toast when update/create for automation fails